src/parser/token.h

```diff
@@ -85,6 +86,9 @@ enum TokenType {
   Extern,
   Match,
   PartialMatch,
+  Metadata,
```

Comment:
Use enum class if possible. Also, I think in Google style people prefix constants with a "k", e.g. kMetadata, kVersion

Reply:
+1